xsl/pretext-html.xsl

```diff
@@ -9693,6 +9695,16 @@ along with MathBook XML.  If not, see <http://www.gnu.org/licenses/>.
     </iframe>
 </xsl:template>
 
+<!-- For more complicated interactives, we just point to the page we generate -->
+<xsl:template match="interactive[@platform='doenetml']" mode="iframe-interactive">
+    <div>
+        <xsl:apply-templates select="." mode="html-id-attribute"/>
+        <xsl:apply-templates select="." mode="permid-attribute"/>
+        <xsl:apply-templates select="." mode="size-pixels-attributes" />
+        <xsl:apply-templates match="slate[@surface = 'doenetml']"/>
+    </div>
+</xsl:template>
+
```

Comment:
Here we overwrite the template that creates an iframe and just render the HTML itself. But this breaks asset generation for capturing preview screenshots - the headless browser explicitly looks for an iframe element, which it doesn't find due to this hack.